### PR TITLE
Remove erroneous STP label from dry air density.

### LINKED ISSUE #54: dry air density is not at STP
The dry air density variables used in the `state_converters` and `kessler` physics schemes have standard names that imply they are at Standard temperature and Pressure (STP).  However, that density quantity is a function of the atmospheric temperature and pressure, which means it is not in fact at STP.  Thus this label should be removed in order to avoid a name collision with density variables that are in fact at STP.

### PR DESCRIPTION
**Pull request by nusbaume**
See title.

Fixes #54

### CODE REVIEW

**Review by peverwhee — APPROVED** [review]

**Comment by cacraigucar** [comment]
An email from @gold2718 :

While I updated the dimension names in the wiki, I am not sure all the variables are there and/or correct. For the most part, I prefer the names used in the Kessler suite. For consistency I suggest the following changes to Kessler:

dry_air_density                                 ==> density_of_dry_air_at_STP

I'm not sure why he chose this and if he has an opinion on our proposed fix

**Review by cacraigucar — APPROVED** [review]

**Comment by nusbaume** [comment]
@gold2718 Since you were the one who made the original naming decision I decided to add you as a reviewer, just to be completely safe (even though I seem to be failing on the "reduce your workload" front).

File: kessler/kessler.meta [hunk 1]
```diff
@@ -168,7 +168,7 @@
   type = integer
   intent = in
 [ rho ]
-  standard_name = density_of_dry_air_at_stp
+  standard_name = density_of_dry_air
```
Comment (gold2718) [inline kessler/kessler.meta:171]: Following the Standard Names dictionary (https://github.com/ESCOMP/CCPPStandardNames/blob/main/Metadata-standard-names.md), this should probably be: `dry_air_density`

File: utilities/state_converters.meta [hunk 2]
```diff
@@ -187,7 +187,7 @@
   dimensions = (horizontal_loop_extent, vertical_layer_dimension)
   intent = in
 [ rho ]
-   standard_name = density_of_dry_air_at_stp
+   standard_name = density_of_dry_air
```
Comment (gold2718) [inline utilities/state_converters.meta:190]: Following the Standard Names dictionary (https://github.com/ESCOMP/CCPPStandardNames/blob/main/Metadata-standard-names.md), this should probably be: `dry_air_density`

**Review by gold2718 — CHANGES_REQUESTED** [review]
(no, I have no idea what I was thinking two years ago when I sent that email).

File: kessler/kessler.meta (hunk 1 above)
Comment (nusbaume) [reply kessler/kessler.meta]: Good call!  Name changed.

File: utilities/state_converters.meta (hunk 2 above)
Comment (nusbaume) [reply utilities/state_converters.meta]: Name changed.

**Review by gold2718 — APPROVED** [review]
:+1:

**Pull request merged**